metadata.yaml

```diff
@@ -24,6 +24,8 @@ containers:
 provides:
```

Comment:
I expected this PR to also include a provided `metrics-endpoint` relation using the `prometheus_scrape` relation interface, so that Grafana Agent can monitor Prometheus (scraping the same endpoints Prometheus scrapes). Having Grafana Agent scrape Prometheus is more resilient in terms of Prometheus failures than Prometheus monitoring itself alone, e.g., when the Prometheus pod is in a crash loop.